### DISCUSSION TITLE
Why can't buy CoPilot Business for myself?

### DISCUSSION THREAD

**Discussion by techdragon** [community/community#51850] [Copilot Conversations]
### Select Topic Area

Question

### Body

I wan't to buy a CoPilot license, but as a freelancer I need that valuable Business grade privacy, _this MATTERS to me and I'm not prepared to pay for CoPilot without the business grade privacy_ ... So I happily go to pay Github extra for the privilege, but weirdly I can't? I have to buy it for an organisation? So is it really this hard? I have to setup a new org, sign it up for business, get the GitHub Copilot for the new org, opt myself in ... etc... 

ALL just to get... me and only me... CoPilot... with the Business Grade Privacy that my personal freelancing contract customers expect me to hold for all their code across all their organisations...

Have I got this correct? I'll probably cave and eventually do this stupid dog and pony show to try CoPilot (I'm a human and have stuff to do after all)... but for now If this is indeed how convoluted it is, I don't want to pay in order to provide the incentive to do better by your customers.

**Comment by bisultankishiev** [comment]
Yeah, GitHub makes [Adxfreework](http://adxfreework.com) harder for freelancers, business privacy shouldn’t need an org setup.
